packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -490,6 +490,10 @@ class Chromium {
         if (i == attempts) {
           rethrow;
         }
+      } on IOException catch (_) {
```

Comment:
this looks weird, is the analyzer mad if this line is just: `} on IOException {` without the catch?

Reply:
addressed in [don't use catch(_)](https://github.com/flutter/flutter/pull/153978/commits/3f12eba91996e497918a61be55444e43c83b4005)